packages/rs-dpp/src/data_contract/document_type/document_field/mod.rs

```diff
@@ -16,130 +16,149 @@ use rand::rngs::StdRng;
 use rand::Rng;
 use serde::{Deserialize, Serialize};
 
+// TODO Must be versioned enum
```

Comment:
Explain why we do not version it and how we gonna version it (it will be a new version of document type)